lib/solReleases.js

```diff
@@ -0,0 +1,40 @@
+const map = require('../lib/mapping.json');
+const rp = require('request-promise');
+
+function getCompilerVersion(parsedData) {
+    return new Promise(async function(resolve, reject){
+        if(parsedData[0].hasOwnProperty('start_version') == false) {
```

Comment:
It will be better to check the statement type for this like `parsedData[0].type == 'PragmaStatement'`